hal/include/HAL/CAN.h

```diff
@@ -11,6 +11,7 @@
 
 #include "FRC_NetworkCommunication/CANSessionMux.h"
 
+extern "C" {
 void canTxSend(uint32_t arbID, uint8_t length,
                int32_t period = CAN_SEND_PERIOD_NO_REPEAT);
```

Comment:
In addition, all of these must be implemented entirely by NI, as the implementations are not in the HAL. So we are not sure if they are extern "C", and I would guess they are not.